packages/front-end/src/app/pages/labs/[labId]/[workflowId]/index.vue

```diff
@@ -70,13 +76,7 @@
     return stoppedDate && stoppedTime ? `${stoppedTime} ⋅ ${stoppedDate}` : '—';
   });
 
-  onBeforeMount(async () => {
-    await initData();
-
-    let paramTab = $router.currentRoute.value.query?.tab;
-    if (!paramTab) paramTab = 'Run Results'; // fallback for no query param to default to first tab
-    tabIndex = paramTab ? tabItems.findIndex((tab) => tab.label === paramTab) : 0;
-  });
+  onBeforeMount(initData);
```

Comment:
Should this by async/await?

Reply:
it should be awaited - this is a simplification of this:

```
onBeforeMount(async () => {
  await initData();
});
```

which to my typescript knowledge is a redundant anonymous function around initData and `onBeforeMount(initData)` is functionally identical.

Reply:
Good to know!